consensus/types/src/chain_spec.rs

```diff
@@ -777,7 +777,7 @@ impl ChainSpec {
             domain_sync_committee_selection_proof: 8,
             domain_contribution_and_proof: 9,
             altair_fork_version: [0x01, 0x00, 0x00, 0x64],
-            altair_fork_epoch: Some(Epoch::new(256)),
+            altair_fork_epoch: Some(Epoch::new(512)),
```

Comment:
Funnily enough this value was already wrong in the `ChainSpec::gnosis` function, which I think indicates that we don't really use these values. I think we'll always (?) override them with the ones from the config.yaml.

Reference for 512 being the correct val: https://github.com/gnosischain/configs/blob/6af0c780f531358ff285cd2807915f1c4caca9ab/mainnet/config.yaml#L150

Reply:
Removing this level of redundancy is out of scope for this PR, so I think we may as well just set them all to the right values